_posts/2020-04-16-remote-junior-developer-tips.markdown

```diff
@@ -0,0 +1,57 @@
+
+Fortunately OmbuLabs made the journey a smooth and easy process. Here are some of the things that have been working.
+
+## Communication, communication, communication.
+
+Communication is sometimes the hardest skill to excel in when working remotely. Here at OmbuLabs we use a variety of tools and methods to help make communication easier and more accessible.
+
+### Slack:
+Communicating on Slack throughout the day is the quickest way to reach the team. As a Junior Developer it can be scary to post code and ask for help. You are constantly walking a fine line between figuring things out on your own so as not to take too much senior developer time, and knowing when it would be faster to ask for help.
+
+A great way to conquer this is to set time limits and guidelines for yourself. When you come up against an error or general topic that you don’t understand, give yourself X amount of time to decipher it. Set a timer, and when that time is up it’s the moment to turn to Slack.
+
+We have many channels on Slack. Choosing the appropriate channel helps to let the right set of Developers know that you have a question. For example, we have a general “learning” channel, but also separate channels for each project.
+
+Sometimes you can just choose a specific Senior Developer to ask the question to. Maybe the Senior Developer has helped you with that area in the past, and you won’t have to re-explain the problem, therefore it can be a quick question.
+
+### Pivotal Tracker:
+We use pivotal tracker to keep track of user stories, and set sprints for the week. Pivotal Tracker also allows for comments and this has become a useful tool when keeping track of what’s been worked on in a story with a Senior Developer.
+
+You can explain the progress that has been made on a story so far. Then, when you  need to ask a Senior Developer for help, you can point them directly to the story which already holds information on the problem.
+
+### Pairing:
+Pairing is the most important tool available to a remote Junior Developer. At OmbuLabs we have a ratio of six Senior Developers to one Junior Developer at the moment. This has allowed for a robust paring schedule. Each senior developer gives two hours of time every other week, that way the Junior Developer receives a total of 6 pairing hours a week. This is invaluable time in the learning process.
+
+## Tracking Progress
+
+On the journey to becoming a Developer there is A LOT to learn. Even when you finally make it to a comfortable place in your Developer career there will still be so much more to learn, so keeping track of it all is very helpful in the process.
+
+### Journal:
+Keeping a journal (I use a google doc) is an important step in organizing all the things you learn or learn that you should learn. During pair sessions and throughout the day it is a good idea to take notes and jot down various resources and ideas that are thrown your way.
```

Comment:
`all the things you learn or learn that you should learn.`

@FionaDL I think there's an extra "learn` there


```suggestion
Keeping a journal (I use a google doc) is an important step in organizing all the things you learn or that you should learn. During pair sessions and throughout the day it is a good idea to take notes and jot down various resources and ideas that are thrown your way.
```